### PR TITLE
Develop abc

### PR DESCRIPTION
**Pull request by rettinghaus**
* fix MEI output problem with `<meterSig>`
* add possibility to exclude ABC import from build
* minor code improvements

### CODE REVIEW

File: include/vrv/ioabc.h [hunk 1]
```diff
@@ -68,10 +68,10 @@ class ABCInput : public Input {
...
     void parseInstruction(std::string keyString); // I:
     void parseKey(std::string keyString); // K:
-    void parseUnitNoteLength(std::string unitNoteLength); // L:
-    void parseMeter(std::string meterString); // M:
-    void parseTempo(std::string tempoString); // Q:
-    void parseReferenceNumber(std::string referenceNumberString); // X:
+    void parseUnitNoteLength(const std::string unitNoteLength); // L:
+    void parseMeter(const std::string meterString); // M:
+    void parseTempo(const std::string tempoString); // Q:
+    void parseReferenceNumber(const std::string referenceNumberString); // X:
```
Comment (lpugin) [inline include/vrv/ioabc.h:74]: Why not as reference?

Comment (rettinghaus) [reply include/vrv/ioabc.h]: Of course!

File: include/vrv/ioabc.h [hunk 2]
```diff
@@ -58,27 +58,27 @@ class ABCInput : public Input {
...
-    void parseTempo(std::string tempoString); // Q:
-    void parseReferenceNumber(std::string referenceNumberString); // X:
+    void parseUnitNoteLength(const std::string &unitNoteLength); // L:
+    void parseMeter(const std::string &meterString); // M:
+    void parseTempo(const std::string &tempoString); // Q:
+    void parseReferenceNumber(const std::string &referenceNumberString); // X:
 
     // input functions
-    void readInformationField(char dataKey, std::string dataValue);
+    void readInformationField(const char &dataKey, std::string dataValue);
```
Comment (lpugin) [inline include/vrv/ioabc.h:77]: And the second string here?

Comment (rettinghaus) [reply include/vrv/ioabc.h:77]: it's called this way:
`           readInformationField(abcLine[0], &abcLine[2]);`

**Pull request merged**